lib/jsdom/living/parent-node.js

```diff
@@ -14,13 +16,14 @@ module.exports = function (core) {
     defineGetter(Constructor.prototype, "children", function () {
       if (!this._childrenList) {
         const self = this;
-        this._childrenList = new core.HTMLCollection(this, function () {
+        this._childrenList = createHTMLCollection(this, function () {
           return domSymbolTree.childrenToArray(self, {filter: function (node) {
             return node.nodeType === NODE_TYPE.ELEMENT_NODE;
           }});
         });
       }
-      this._childrenList._update();
+
+      updateHTMLCollection(this._childrenList);
```

Comment:
`create` already does update. Put it into an `else` to avoid building the collection twice? Same deal with NodeCollections.